### PR TITLE
Add telemetry to opni manager

### PR DESCRIPTION
**Pull request by dbason**
Adds the upgrade checker/telemetry in to the Opni controller manager.  Also minor tidyups for Tilt.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -42,6 +44,11 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+	Version  = "v0.1.0-dev"
```
Comment (kralicky) [inline main.go:47]: should this just be "dev" do you think? this is the version that would show up if `go get` was used to download the binary.

Comment (dbason) [reply main.go:47]: Sure thing, I initially had it this way to test the semver comparison.

Comment (kralicky) [reply main.go]: should we disable update checking if the verison is dev?

File: main.go [hunk 2]
```diff
@@ -42,6 +44,11 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+	Version  = "dev"
```
Comment (tybalex) [inline main.go:47]: this would be the release version number , right? eg.. `v0.1.3`? do we have to manually change this value for every release?

Comment (dbason) [reply main.go:47]: Nope we set it using ldflag when we build

File: scripts/manager [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
-cd $(dirname $0)/..
+cd $(dirname $0)
 
-CGO_ENABLED=0 go build -ldflags '-w -s' -o bin/manager .
\ No newline at end of file
+source version
+
+cd ..
+
+CGO_ENABLED=0 go build -ldflags "-w -s -X main.Version=$VERSION" -o bin/manager .
```
Comment (dbason) [inline scripts/manager:9]: @tybalex this line here is where we set the version at build time

**Review by kralicky — APPROVED** [review]

**Pull request merged**